### PR TITLE
Add indexing env vars

### PR DESCRIPTION
**Pull request by rsevilla87**
Add indexing related env vars in order to make ES indexing more configurable from the command line.

cc: @mukrishn @josecastillolema @smalleni 

Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### CODE REVIEW

**Review by mukrishn — APPROVED** [review]

File: workload/cfg_delete_icni1_f5_cluster_density.yml [hunk 1]
```diff
@@ -5,9 +5,9 @@ global:
   metricsDirectory: collected-metrics
   indexerConfig:
     enabled: false
-    esServers: [https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com]
+    esServers: {{ .ES_SERVER }}
     insecureSkipVerify: true
-    defaultIndex: kube-burner
+    esServers: {{ .ES_INDEX }}
```
Comment (smalleni) [inline workload/cfg_delete_icni1_f5_cluster_density.yml:10]: Shouldn't this be defaultIndex

File: workload/cfg_delete_icni1_f5_node_density.yml [hunk 2]
```diff
@@ -5,9 +5,9 @@ global:
   metricsDirectory: collected-metrics
   indexerConfig:
     enabled: false
-    esServers: [https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com]
+    esServers: {{ .ES_SERVER }}
     insecureSkipVerify: true
-    defaultIndex: kube-burner
+    esServers: {{ .ES_INDEX }}
```
Comment (smalleni) [inline workload/cfg_delete_icni1_f5_node_density.yml:10]: defaultIndex

File: workload/cfg_delete_icni2_cluster_density2.yml [hunk 3]
```diff
@@ -5,9 +5,9 @@ global:
   metricsDirectory: collected-metrics
   indexerConfig:
     enabled: false
-    esServers: [https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com]
+    esServers: {{ .ES_SERVER }}
     insecureSkipVerify: true
-    defaultIndex: kube-burner
+    esServers: {{ .ES_INDEX }}
```
Comment (smalleni) [inline workload/cfg_delete_icni2_cluster_density2.yml:10]: defaultIndex

**Review by smalleni — CHANGES_REQUESTED** [review]
Seems like there was an accidental replacement of defaultIndex with esServers?

File: create_icni2_workload.sh [hunk 4]
```diff
@@ -8,6 +8,9 @@ export QPS=${QPS:-20}
 export BURST=${BURST:-20}
 export SCALE=${SCALE:-1}
 export BFD=${BFD:-false}
+export INDEXING=${INDEXING:-true}
+export ES_SERVER=${ES_INDEX:-https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com}
```
Comment (smalleni) [inline create_icni2_workload.sh:12]: Also ES_SERVER

**Comment by rsevilla87** [comment]
> Seems like there was an accidental replacement of defaultIndex with esServers?

Fixed, thanks for noticing it

**Pull request merged**